### ISSUE TITLE
Incorrect value for media resource property in Ithomids.pld

### ISSUE THREAD

**Issue by lowery** [electronicfieldguide/efg-keys#4]
Image "Gody-zyg-md-663-184.jpg" doesn't exist but "Gody-zygi-md-551-1-184.jpg "does (same as on the field guide species page) located in images/EFGImages.

See: https://github.com/electronicfieldguide/efg-keys/blob/5f068c9/pld/Ithomids.pld#L1945

**Comment by ramorrismorris** [comment]
http://bdei2.cs.umb.edu:8080/efg2/Redirect.jsp?displayFormat=html&dataSourceName=itho_1282360908575&ALL_TABLE_NAME=efg_rdb_tables&uniqueID=1282360908591 may show where the problem arose in the database access.   Also,  I think the "663" denotes width (or height) in pixels. If so the proposed substitute  is too small in one dimension.  Also I'm suspicious of the spelling which, on the html page for Godyris zygia  the missing image is  http://bdei2.cs.umb.edu:8080/efg2/efgimagesthumbs/Ith/God-zyg-mv-663.jpg  whose local part probably also should be .../Gody-zygi-md...

We can check with Bill of we can't find a plausible jpg of the right size.

**Comment by lowery** [comment]
@ramorrismorris Yes I think that you're right about the size of the images. Also I think that the mv and md parts of the filename are referring to male ventral or male dorsal? Will hold off on this one to see what Bill says.
